### ISSUE TITLE
Keyless drive function not available 

### ISSUE THREAD

**Issue by fritzkleiner** [Kemmey/Kemmey-TeslaWatch-Public#170]
I can‘t use the keyless drive function on my apple watch. The function is grey and not available in the watch for tesla  app.

**Comment by Kemmey** [comment]
Did you activate it under settings and input your password in the text field below?

**Comment by fritzkleiner** [comment]
Yes, I‘m logged in. I got all model 3 information from my Tesla. Only keyless drive and charging port unlock is grey. All other functions are available.

With kind regards,
Fritz


Fritz Kleiner
Kirchbergstrasse 4
8635 Dürnten
Kleiner.fritz@yahoo.com

**Comment by Kemmey** [comment]
But have you activated remote start from settings and input your password in the text field?

![image](https://user-images.githubusercontent.com/4688084/95627530-a7100700-0a7c-11eb-986e-65bceb0c6fde.png)

**Comment by fritzkleiner** [comment]
Sorry, but I can‘t see/find this function. I use the german language of the appl.
I use apple watch 6 with version 7.0.1
I can also not find the watch screen of your app.
Based on the app screen picture on your site https://www.watchfortesla.com/ , I can not finde the first screen (watch design) and the last screen (keyless drive).
Please see picture attached.

Thank you for your help.
With kind regards,
   Fritz 
Fritz Kleiner
Kirchbergstrasse 4
8635 Dürnten
Kleiner.fritz@yahoo.com

**Comment by Kemmey** [comment]
For the Remote start (renamed from Keyless drive, as that name was confusing to some), you scroll to the bottom of the main screen, and the button at the bottom is "Settings" / "Einstellungen". Tap that and then scroll down to "Remote start" / "Schlüsselloses Fahren" and tap the toggle to turn it on. Then carefully read the disclaimer text, then tap in the text input field below the disclaimer text and input your Tesla account password. As the disclaimer text mentions, for the remote start feature to work, the Tesla API needs your password on every request. So you need to enter it into the app so the app knows your password. This is the only feature that require your password to be known - all other features do not - this is why remote start is an optional feature, that you may enable if you trust the app with your password. 

At this point, Remote start should be operational, as long as you can already do it from the Tesla iOS app - it also has to be enabled in the vehicle etc.

**Comment by fritzkleiner** [comment]
Thank you Kim

No, it works :-)
A short description would be very helpful for other customers, to get it work.

With kind regards from Switzerland,
  Fritz

Fritz Kleiner
Kirchbergstrasse 4
8635 Dürnten
Tel. +41552412023
Mobil +41794031145
Kleiner.fritz@yahoo.com <mailto:Kleiner.fritz@yahoo.com>
Buch Link: IT Service Management - Aus der Praxis für die Praxis (MITP 2016) <https://www.mitp.de/BUSINESS-MARKETING/Business/IT-Service-Management.html?listtype=search&searchparam=kleiner>
Buch Link: IT Financial Management - Aus der Praxis für die Praxis (MITP 2019) <https://www.mitp.de/BUSINESS-MARKETING/Projektmanagement/IT-Financial-Management.html?listtype=search&searchparam=kleiner>
